ui/pages/settings/index.js

```diff
@@ -274,6 +274,22 @@ export default function Settings() {
                           <div className='truncate text-sm font-medium text-gray-700'>
                             {info.getValue()}
                           </div>
+                          {info.row.original.created && (
+                            <div className='space-y-1 pt-2 text-3xs text-gray-500 sm:hidden'>
+                              created -{' '}
+                              <span className='font-semibold text-gray-700'>
+                                {moment(info.row.original.created).from()}
+                              </span>
+                            </div>
+                          )}
+                          {info.row.original.lastUsed && (
+                            <div className='space-y-1 pt-2 text-3xs text-gray-500 sm:hidden'>
+                              last used -{' '}
+                              <span className='font-semibold text-gray-700'>
+                                {moment(info.row.original.lastUsed).from()}
```

Comment:
Do we know if `lastUsed` is always set? This may cause an error in `moment`

Reply:
this is only on the small screen and this wont random if `info.row.original.lastUsed` is not true